.changeset/tiny-chicken-worry.md

```diff
@@ -0,0 +1,5 @@
+---
+"xstate-viz-app": patch
+---
+
+Added SSR and OG image tags to visualised machines, and put visualised machines on a new, prettier URL - `/viz/:id`
```

Comment:
All 3 things are completely separate and unrelated from the user's PoV. Don't be afraid of creating more than a single changeset per PR :)

I'm not sure if SSR is that interesting to our users - but the other ones definitely are! A sample OG image could also be attached to a changeset mentioning this improvement.